src/atoms/TextInput/TextInput.native.js

```diff
@@ -249,6 +249,10 @@ const TextInput = ({
     [layoutDimensions, setLayoutDimensions],
   );
 
+  useEffect(() => {
+    if (!isEmpty(children)) setInput(children);
```

Comment:
```suggestion
    setInput(children);
```

Reply:
Why are you ignoring the empty case? If you want to clear the text field then?

Reply:
You're right. Let me rectify this.